raiden/ui/cli.py

```diff
@@ -204,6 +204,13 @@ def options(func):
                 type=str,
                 show_default=True,
             ),
+            option(
+                '--pathfinding-service-address',
+                help='"host:port" for the raiden pathfinding service to request paths.',
+                type=str,
+                default='127.0.0.1:6000',
```

Comment:
Also, AFAIK the PFS provides a REST API, that would mean this value should be a URL instead of a plain `<ip>:<port>` pair.

Reply:
This is a good point, I will re-check. 
What about: the default value should be always the PFS that the user used the last time actually, at the first time default should be the localhost in case, the user does not want to use and pay for a PFS

Reply:
We can't really provide a sensible default at this point. Once we have the RSB registry of third party services, then we could pick one of those, but for now a local host is the only choice of a default I can imagine.

@Dominik1999 's suggestion of remembering the last choice is also good.

Reply:
I can open an issue for redefining / optimizing the default value in the cli argument. Then you could accept this PR and we don't forget to optimize. Would that work?

Reply:
Fine by me. @ulope ?

Reply:
Yes that's fine, but I'd say just remove the default for now. There is really no benefit in having it AFAICS.

Reply:
ok default is removed for now